packages/core/admin/ee/admin/content-manager/pages/ListView/ReviewWorkflowsColumn/constants.js

```diff
@@ -39,7 +39,13 @@ export const REVIEW_WORKFLOW_COLUMNS_EE = [
         defaultMessage: 'Assignee',
       },
       searchable: false,
-      sortable: false,
+      sortable: true,
+      mainField: {
+        name: 'firstname',
```

Comment:
Hum, does that make sense? Might be good to check with Paridhi how we want to sort?

Are you planning on adding the assignee to the default sort in this branch too? It might make more sense to do that on top of `feature/rw-stage-default-sort` because it contains everything we need for that BE side.

Reply:
Yeah I'll check, you mean does it makes sense to sort by `firstname`? 

Yeah I was wondering about that. If we merged this and then I worked on default sort separately from https://github.com/strapi/strapi/pull/17477 would that make sense?

Reply:
Hmmm that sounds a bit tricky as we can only define one `mainField` right? and no matter what we pick there will be cases where that field is not visible?

Reply:
Yes - I don't think we can make this dynamic which is also a reason why I think we should discuss it with Paridhi and maybe @Marc-Roig to understand which case makes the most sense.

Reply:
Are we going to use the `getDisplayName()` util that content studio implemented? (cc @Feranchz  in case you want to join)

What do we sort by for creator fields? Bcause imo we should use the same.

Reply:
@Feranchz Thanks - I totally agree.

@Marc-Roig mentioned earlier we might be able to sort on multiple fields. Could you elaborate on how that would work?

I'm wondering if we could leave it as it is, but we highlight the problem to product/ design to improve it in parallel? E.g. letting the user choose on which field to sort or whatever else comes to mind :)

Reply:
Letting user choose field could be a good idea indeed,

About multiple sorting, this is something you can do in strapi:
`sort[username]&sort[firstname]` will give priority to username, and in case it is null it will sort by first name.

I also think this could be a bit problematic for the defaultSort configuration, as you can not store more than one defaultSort , and changing that would make things a bit more complex.

Reply:
@jhoward1994 Since Fernando's work is already done: would you maybe try out what the effects of using the multiple syntax would be? From the theory it sounds like the perfect use case to me: username has precedence over lastname over firstname.

Reply:
New PR to allow sorting on multiple relational fields: https://github.com/strapi/strapi/pull/17499